airbyte-webapp/src/views/Connector/ServiceForm/ServiceForm.test.tsx

```diff
@@ -268,8 +268,9 @@ describe("Service Form", () => {
       const password = container.querySelector("input[name='connectionConfiguration.password']");
       const message = container.querySelector("textarea[name='connectionConfiguration.message']");
       const apiKey = container.querySelector("input[name='connectionConfiguration.credentials.api_key']");
-      const emails = container.querySelector("input[name='connectionConfiguration.emails']");
       const workTime = container.querySelector("div[name='connectionConfiguration.workTime']");
+      const emails = screen.getByTestId("tag-input").querySelector("div > div > div > div > input");
```

Comment:
I really dislike this selector.  If anyone is able to find a better one, I would love to use it.  Tried a number of different methods (getBys, searching for the div(s) by partial classnames, etc.) before settling for this.  It does work, though.